astro.config.mjs

```diff
@@ -27,13 +27,17 @@ const remarkEmbedPlugin = [remarkEmbedder.default, {
   }
 }];
 
+const { IS_PROD_BUILD } = process.env
 
 // https://astro.build/config
 export default defineConfig({
-  site: 'https://okturtles.github.io',
-  base: '/groupincome.org',
+  site: IS_PROD_BUILD === 'true' ? "https://our_prod_website_url.org" : 'https://okturtles.github.io',
```

Comment:
Is this `https://our_prod_website_url.org` supposed to be here? If so could you add a comment explaining why it's here and if it needs to be overwritten, how to do so?